client/state/current-user/gravatar-status/selectors.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
+
+/**
+ * Returns true if we're currently uploading a Gravatar
+ * @function
```

Comment:
No need for `@function` since there should be no confusion to parser here that it is one.